pkg/config/config.go

```diff
@@ -84,15 +84,20 @@ func (c *KVConfiguration) GetReplications() (map[constants.AvailabilityZone][]st
 			storeLatency = latencyResult.Summary.Success.Average / 1000000
 		}
 
+		if c.StoreType != constants.Redis {
```

Comment:
suggest to treat all backend types same here. If some db types need e.g. ip:port, they can always get by looking up database.Storages map

Reply:
As we discussed, it will be used for getting latencies.